docs/3.0.0-beta.x/guides/jwt-validation.md

```diff
@@ -0,0 +1,103 @@
+# JWT validation
+
+In this guide we will see how validate a JWT via another service.
+
+When you signin with the authentication route `POST /auth/local`, Strapi generate a `JWT` that let your users request your API as an authenticated user.
+
+```json
+{
+  "jwt": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiaXNBZG1pbiI6dHJ1ZSwiaWF0IjoxNTcxODIyMDAzLCJleHAiOjE1NzQ0MTQwMDN9.T5XQGSDZ6TjgM5NYaVDbYJt84qHZTrtBqWu1Q3ShINw",
+  "user": {
+    "email": "admin@strapi.io",
+    "id": 1,
+    "username": "admin"
+  }
+}
+```
+
+These users are managed in the application database and can be managed via the admin dashboard.
+
+We can now imagine you have a `JWT` that come from [Auth0](https://auth0.com) and you want to make sure the `JWT` is correct and allow this user to use the Strapi API endpoints.
+
+## Customise the JWT validation function
```

Comment:
Like we chose to use `behavior` (the US version) instead of `behaviour` (the English) one I think it should be better to use `customize` 

> In the U.S. and Canada, it’s customize, customized, customizing, customization, customizable, etc. Outside North America, it’s customise, customised, customising, customisation, customisable, etc.